### PR TITLE
Documenting browser support

### PR DESCRIPTION
**Pull request by jimmynotjim**
As part of cfpb/development#134 this PR aims to document the current level of browser support within this project.

### CODE REVIEW

**Comment by jimmynotjim** [comment]
Seems I can't assign reviewers. @mistergone and @sebworks mind reviewing this when you have the time?

**Review by sebworks — APPROVED** [review]

**Comment by jimmynotjim** [comment]
Looks like the build is continuing to fail even though it's a docs change. Anyone from the team able to take a look at it?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,3 +30,61 @@ tests that validate implemented features and the presence or lack of defects.
...
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
+    - Opera Mobile
+    - Chrome for Android
+    - Firefox for Android
+    - Samsung Internet
+- Internet Explorer 9
+- Internet Explorer 8
```
Comment (anselmbradford) [inline CONTRIBUTING.md:54]: Project has reference to IE7 https://github.com/cfpb/college-costs/blob/master/gulp/tasks/styles.js#L40

**Comment by chosak** [comment]
@jimmynotjim I think this Travis failure has been fixed in a subsequent PR in 9d98462. Can you rebase this PR (I can't do that on your branch) and force push to retrigger Travis?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jimmynotjim) [reply CONTRIBUTING.md:54]: @anselmbradford this has been addressed.

**Comment by jimmynotjim** [comment]
@chosak thanks for the heads up, rebased and pushed.

**Review by anselmbradford — APPROVED** [review]

**Comment by jimmynotjim** [comment]
@chosak Do you have merge permissions? I can't merge this one myself.

**Comment by chosak** [comment]
@jimmynotjim I do! Thank you for rebasing.

**Pull request merged**